pkg/webhook/validators.go

```diff
@@ -0,0 +1,56 @@
+package webhook
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	cv1alpha1 "github.com/openservicemesh/osm/pkg/apis/config/v1alpha1"
+	pv1alpha1 "github.com/openservicemesh/osm/pkg/apis/policy/v1alpha1"
+	admissionv1 "k8s.io/api/admission/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func init() {
+	gvk := metav1.GroupVersionKind{
+		Kind:    "Egress",
+		Group:   "policy.osm.io",
+		Version: "v1alpha1",
+	}
+	RegisterValidator(gvk.String(), EgressValidator)
+	RegisterValidator(gvk.String(), DurationValidator)
```

Comment:
This should be the MeshConfig gvk

Reply:
I made 2 gvks. One for egress and another for meshconfig. Does that look right?